cap-2.x/gke/main.tf

```diff
@@ -0,0 +1,65 @@
+module "gke-cluster" {
+  source = "./cluster"
+
+  location = var.location
+  cluster_labels = var.cluster_labels
+  instance_count = var.instance_count
+  instance_type = var.instance_type
+  disk_size_gb = var.disk_size_gb
+  dns_credentials_json = var.dns_credentials_json
+}
+
+// kubeconfig will be generated by the GKE module
+
+provider "helm" {
+  version = "1.3.2"
+  alias   = "helm-cap"
+
+  kubernetes {
+    config_path = "${path.cwd}/kubeconfig"
+  }
+}
+
+module "helper-charts" {
+  source = "./common/helper-charts"
+
+  providers = {
+    helm = helm.helm-cap
+  }
+
+  platform             = "gke"  //for common helper-charts' tmpl files
+
+  service_provider     = "google"
+
+  project              = var.project
+  dns_credentials_json_secret_key_name = module.gke-cluster.dns_credentials_json_secret_key_name
+
+  email                   = var.email
+
+  depends_on = [
+    module.gke-cluster
+  ]
+}
+
+module "cap-charts" {
+  source = "./common/cap-charts"
+
+  providers = {
+    helm = helm.helm-cap
+  }
+
+  cap_domain = "${module.gke-cluster.cluster_name}.${var.cap_domain}"
```

Comment:
Should be `cap_domain = var.cap_domain`

The UAA url doesn't have <cluster-name>.uaa.<domain> so the url above fails to connect from `wait_for_uaa.sh`

Reply:
done https://github.com/SUSE/cap-terraform/pull/97/commits/de575001c40a56e125ae03a793101678e089b42d

Reply:
Clarification - the end-user would need to choose a unique subdomain, not the implementation (e.g. cluster name). Hence, no need for cluster name in front of the chosen domain.